---
layout: post
title:  Chromosome-scale genome assembly and population genomics provide insights into the adaptation,domestication, and flavonoid metabolism of Chinese plum
cover-img: /assets/img/P7-1.jpg
thumbnail-img: /assets/img/P7.jpg
share-img: /assets/img/P7-3.jpg
tags: [plum, plum]
---

Globally, commercialized plum cultivars are mostly diploid Chinese plums (Prunus salicina Lindl.), also known as Japanese plums, and are one of the most abundant and variable fruit tree species. To advance Prunus genomic research, we present a chromosome-scale P. salicina genome assembly, constructed using an integrated strategy that combines Illumina, Oxford Nanopore, and high-throughput chromosome confor-mation capture (Hi-C) sequencing. The high-quality genome assembly consists of a 318.6-Mb sequence (con-tig N50 length of 2.3 Mb) with eight pseudo-chromosomes. The expansion of the P. salicina genome is led by recent segmental duplications and a long terminal repeat burst of approximately 0.2 Mya. This resulted in a significant expansion of gene families associated with flavonoid metabolism and plant resistance, which impacted fruit flavor and increased species adaptability. Population structure and domestication his-tory suggest that Chinese plum may have originated from South China and provides a domestication route with accompanying genomic variations. Selection sweep and genetic diversity analysis enabled the identifi-cation of several critical genes associated with flowering time, stress tolerance, and flavonoid metabolism, demonstrating the essential roles of related pathways during domestication. Furthermore, we reconstructed and exploited flavonoidâ€“anthocyanin metabolism using multi-omics analysis in Chinese plum and proposed a complete metabolic pathway. Collectively, our results will facilitate further candidate gene discovery for important agronomic traits in Chinese plum and provide insights into future functional genomic studies and DNA-informed breeding.