---
layout: post
title:  BSATOS in Journal of Advanced Research
cover-img: /assets/img/Gaphic_00.jpg
thumbnail-img: /assets/img/Gaphic_00.jpg
share-img: /assets/img/Gaphic_00.jpg
tags: [bsatos, bsatos]
---

Welcome to the BSATOS wiki! bulked-segregant analyis tools for outbreeding species(BSATOS) is developed by the institute for horticultue plants, China Argicultural University and Edmund Mach Foundation, Italy. BSATOS is designed for NGS-based Bulked-segregant analysis for outbreeding species, includings fruits trees, such as apple, and cirtus.

To improve the gene mapping efficiency of next generation sequencing-based segregant analysis (BSA) in outbreeding species and realize the rapid candidate gene mining based on multi-omics data, bulked segregant analysis tools for outbreeding species (BSATOS) was developed. Different from the classic two-way pseudo-testcross (PT) strategy, Local phased 3-way BSA mapping strategy was introduced to use three type of segregate makers unbiased. After comparing different statistical methods to evaluate the allele frequency difference, G value method was selected. Multi-round G value screening and multi-omics data based gene mining were also integrated.

Genomic heterozygosity, self-incompatibility, and rich-in somatic mutations hinder the molecular breeding efficiency of outcrossing plants. Objectives: We attempted to develop an efficient integrated strategy to identify quantitative trait loci (QTLs) and trait-associated genes, to develop gene markers, and to construct genomics-assisted prediction (GAP) modes. Methods: A novel protocol, bulked segregant analysis tool for out-crossing species (BSATOS), is presented here, which is characterized by taking full advantage of all segregation patterns (including AB Ã— AB markers) and haplotype information. To verify the effectiveness of the protocol in dealing with the complex traits of outbreeding species, three apple cross populations with 9,654 individuals were adopted. Results: By using BSATOS, 90, 60, and 77 significant QTLs were identified successfully and candidate genes were predicted for apple fruit weight (FW), fruit ripening date (FRD), and fruit soluble solid content (SSC), respectively. The gene-based markers were developed and genotyped for 1,396 individuals in a training population, including 145 Malus accessions and 1,251 F1 plants of the three full-sib families. GAP models were trained using marker genotype effect estimates of the training population. The prediction accuracy was 0.7658, 0.6455, and 0.3758 for FW, FRD, and SSC, respectively. Conclusion: The BSATOS and GAP models provided a convenient and efficient methodology for candidate gene mining and molecular breeding in out-crossing plant species. The BSATOS pipeline can be freely downloaded from: https://github.com/maypoleflyn/BSATOS.





